src/test/cli/rbd/invalid-snap-usage.t

```diff
@@ -89,6 +89,7 @@
   rbd: snapname specified for a command that doesn't use it
   [22]
   $ rbd bench-write foo@snap
+  rbd: bench-write is deprecated, use bench --io-type write
```

Comment:
@yangdongsheng sorry, missed this in my previous review. I think it is better to change this to `rbd bench --io-type write ...`, so don't use deprecated command. The same is for not-enough-args.t below.